input/fsh/omrs-condition.fsh

```diff
@@ -0,0 +1,52 @@
+Profile:      OMRSCondition
+Parent:       Condition
+Id:           omrs-Condition
+Title:        "OpenMRS condition"
+Description:  "A FHIR Condition as understood by OpenMRS"
+* identifier 0..*
```

Comment:
I don't see an Identifier in the fhir2 translator. Also, I don't see support for `clinicalStatus` or `verificationStatus`

Reply:
Resolved Thanks.